.gitattributes

```diff
@@ -0,0 +1 @@
+* text eol=lf
```

Comment:
@roman-petrov how about merge `.gitattributes` in separate PR? Anyway this is are useful changes.

Reply:
sure, this is a good idea! I will do this today

Reply:
Done: https://github.com/dart-code-checker/dart-code-metrics/pull/337